lib/graphql/schema.graphql

```diff
@@ -1499,6 +1514,7 @@ input MemberInputType {
 }
 
 """
+<<<<<<< HEAD
```

Comment:
Seems like you have an unsolved merge conflict on that file

Reply:
Ouch